### DISCUSSION TITLE
A Lot of Error Messages?!

### DISCUSSION THREAD

**Discussion by TrigMan59** [openai/whisper#1425] [Q&A]
I was able to successfully transcribe one MP3 file this afternoon. When I tried a second one, I got all of these error messages. Any recommendations? By the way the filename is correct.

  def backtrace(trace: np.ndarray):
Traceback (most recent call last):
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\lib\site-packages\whisper\audio.py", line 46, in load_audio
    ffmpeg.input(file, threads=0)
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\lib\site-packages\ffmpeg\_run.py", line 325, in run
    raise Error('ffmpeg', out, err)
ffmpeg._run.Error: ffmpeg error (see stderr output for detail)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\Scripts\whisper.exe\__main__.py", line 7, in <module>
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\lib\site-packages\whisper\transcribe.py", line 437, in cli
    result = transcribe(model, audio_path, temperature=temperature, **args)
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\lib\site-packages\whisper\transcribe.py", line 121, in transcribe
    mel = log_mel_spectrogram(audio, padding=N_SAMPLES)
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\lib\site-packages\whisper\audio.py", line 130, in log_mel_spectrogram
    audio = load_audio(audio)
  File "C:\Users\roger\AppData\Local\Programs\Python\Python310\lib\site-packages\whisper\audio.py", line 51, in load_audio
    raise RuntimeError(f"Failed to load audio: {e.stderr.decode()}") from e
RuntimeError: Failed to load audio: ffmpeg version 6.0-essentials_build-www.gyan.dev Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 12.2.0 (Rev10, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-lzma --enable-zlib --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-sdl2 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libaom --enable-libopenjpeg --enable-libvpx --enable-libass --enable-libfreetype --enable-libfribidi --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-libgme --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libtheora --enable-libvo-amrwbenc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-librubberband
  libavutil      58.  2.100 / 58.  2.100
  libavcodec     60.  3.100 / 60.  3.100
  libavformat    60.  3.100 / 60.  3.100
  libavdevice    60.  1.100 / 60.  1.100
  libavfilter     9.  3.100 /  9.  3.100
  libswscale      7.  1.100 /  7.  1.100
  libswresample   4. 10.100 /  4. 10.100
  libpostproc    57.  1.100 / 57.  1.100
02-Regional_Convention_2023.mp3: No such file or directory

Thanks for any and all help.

Roger

**Comment by mrfragger** [comment]
`RuntimeError: Failed to load audio: ffmpeg version 6.0-essentials_  `
I think that's your problem.  Perhaps you have and older version of ffmpeg in your path.

I did this over a year ago.  I believe I encountered some errors and possibly showed when processing with fre:ac but can't say for sure.  But if you do the mp3 to mkv then that should at least correct some errors with mp3s.  Most likely a corrupt mp3.

This error can be corrected with MKVToolnix
Error while decoding stream #0:0: Invalid data found when processing input
Audio packet of size 24 (starting with 00000000...) is invalid, writing it anyway

This error is fatal and the mp3 should be deleted
mp3float @ 0x5639205e7a40] invalid new backstep -1
[mp3float @ 0x5639205e7a40] big_values too big
[mp3float @ 0x5639205e7a40] Error while decoding MPEG audio frame.
Error while decoding stream #0:0: Invalid data found when processing input

Some errors with some mp3s so multiplexing them to mka with MKVToolnix (https://mkvtoolnix.download/) solved the problem. 

To batch multiplex mp3s all at once in MKVToolnix when dragging them into the app choose 
Create new multiplex settings for each file
this will create a tab for each mp3
Click Multiplex then Action for all tabs then Start multiplexing
When done you can Multiplex then Action for all tabs then Close

**Comment by TrigMan59** [comment]
@mrfragger Thanks so much for your help and reply. I greatly appreciate. I re-downloaded the MP3 file and gave it an 8 character name. Instead of running Whisper using the CMD window, I used PowerShell. It worked perfectly this time.

I wish I understood your reference to MKVToolnix. I'm definitely a programming newbie and know very little.

I'm trying to find a Forum where I can enlist someone's help in creating, what I think is a simple program in Python. I need a program to login to a Zoom meeting, start OBS Studio and record the Zoom meeting and when the meeting is finished, stop recording, close OBS and Zoom. There are snippets of code here on GitHub to do some of these things. I just don't have the programming skill set to figure it out. I wish I could afford Co-Pilot for Visual Studio, but being disabled and on a fixed income, I just can't afford it. Thanks again for all of your help.
